dbt_meshify/dbt_projects.py

```diff
@@ -328,11 +328,16 @@ def find_jinja_blocks(self) -> Dict[str, JinjaBlock]:
             )
 
         for unique_id, macro in self.manifest.macros.items():
+            block_type = "macro"
+            name = macro.name
             if macro.package_name != self.name:
                 continue
+            if "tests/generic/" in macro.path:
+                block_type = "test"
+                name = macro.name[5:]
```

Comment:
I don't feel super confident in using this hard-coded path. Would it make sense to use the test directory provided in `dbt_project.yml` here instead?

Reply:
that makes sense to me! will update